core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -3808,7 +3828,6 @@ public void testInsertBinaryBindVariable2() throws Exception {
     }
 
     @Test
-    @Ignore // TODO: support big binary parameter buffers (epic)
     public void testInsertBinaryOver1Mb() throws Exception {
```

Comment:
nit: might be a good idea to add a test for the "undersized receive buffer or someone is abusing protocol" case.

Reply:
ok, added tests with send and recv buffer overflows in https://github.com/questdb/questdb/pull/3751/commits/6b8bc405d52626bc2df0ac722ad80036c4ae7306